notebooks/03_computer_vision_dnn.ipynb

```diff
@@ -150,18 +148,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [
     {
-     "data": {
-      "text/plain": [
-       "<vflow.vset.Vset at 0x7f8c26ab00f0>"
-      ]
-     },
-     "execution_count": 5,
-     "metadata": {},
-     "output_type": "execute_result"
+     "output_type": "error",
```

Comment:
There seems to be a new error here due to the previous changes to `build_vset`; could you please see if changing `NeuralNetwork` to `model=NeuralNetwork` fixes this?